pkgs/applications/misc/sunwait/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+let
+  pname = "sunwait";
+  version = "2020-10-26";
+in stdenv.mkDerivation {
+  inherit pname version;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "sunwait";
  version = "2020-10-26";
```

Reply:
I'm removing the let statement, but I'm not adding the rec because it's not needed.